crates/bevy_solari/src/scene/binder.rs

```diff
@@ -0,0 +1,353 @@
+use super::{blas::BlasManager, extract::StandardMaterialAssets, RaytracingMesh3d};
+use bevy_asset::{AssetId, Handle};
+use bevy_color::LinearRgba;
+use bevy_ecs::{
+    resource::Resource,
+    system::{Query, Res, ResMut},
+    world::{FromWorld, World},
+};
+use bevy_math::{ops::cos, Mat4, Vec3};
+use bevy_pbr::{ExtractedDirectionalLight, MeshMaterial3d, StandardMaterial};
+use bevy_platform::{collections::HashMap, hash::FixedHasher};
+use bevy_render::{
+    mesh::allocator::MeshAllocator,
+    render_asset::RenderAssets,
+    render_resource::{binding_types::*, *},
+    renderer::{RenderDevice, RenderQueue},
+    texture::{FallbackImage, GpuImage},
+};
+use bevy_transform::components::GlobalTransform;
+use core::{hash::Hash, num::NonZeroU32, ops::Deref};
+
+const MAX_MESH_SLAB_COUNT: NonZeroU32 = NonZeroU32::new(500).unwrap();
```

Comment:
Just to double check, this is the number of meshes per slab, and not the total number of meshes in the scene?